Cargo.toml

```diff
@@ -13,11 +13,11 @@ crossbeam = "0.7"
 env_logger = "*"
 err-derive = "0.2"
 glib = "*"
-gio = "0.8"
-gstreamer = "0.15"
-gstreamer-app = "0.15"
-gstreamer-rtsp = "0.15"
-gstreamer-rtsp-server = { version = "0.15", features = ["v1_12", "v1_14"]}
+gio = "0.9"
+gstreamer = "0.16"
+gstreamer-app = "0.16"
+gstreamer-rtsp = "0.16"
+gstreamer-rtsp-server = { version = "0.16", features = ["v1_12", "v1_14", "v1_16"]}
```

Comment:
I recommend using

```toml
gstreamer-rtsp-server = { version = "0.16", features = ["v1_12", "v1_14"]}
```

Instead of
```toml
gstreamer-rtsp-server = { version = "0.16", features = ["v1_12", "v1_14", "v1_16"]}
```

Unless we explicitly need something in the `v1_16` set. The `v1_16` is the version of the gstreamer plugin not the rust binding.

Reply:
That was indeed the problem. Thanks!